Packs/CommonTypes/Layouts/layoutscontainer-ReportV2.json

```diff
@@ -263,11 +265,12 @@
                         "type": "relationshipsTable",
                         "w": 2,
                         "x": 0,
-                        "y": 3
+                        "y": 3,
+                        "maxH": null
                     },
                     {
                         "h": 3,
-                        "i": "vug7e0qild-09aff230-b30b-11eb-941c-e725acf1fbbc",
+                        "i": "vug7e0qild-f8d5a311-442a-42af-89fd-45a8a1b83fff-vug7e0qild-f8d5a311-442a-42af-89fd-45a8a1b83fff-vug7e0qild-09aff230-b30b-11eb-941c-e725acf1fbbc",
```

Comment:
Is that OK? looks odd to me as the new objects were added with shorter `i`

Reply:
Thats a system index, no idea.

Reply:
The format didn't fix that?

Reply:
The format probably made it, it's not an error though...